### DISCUSSION TITLE
Option to return from "add" command as soon as processing starts

### DISCUSSION THREAD

**Discussion by FunctionDJ** [ArchiveBox/ArchiveBox#1006] [Q&A]
Hi, i'm using `archivebox add` but i'd like to return to the terminal as soon as processing of the URLs has started. Otherwise the terminal would be blocked until all media has been downloaded, but i'm only interested in if the parsing of the URLs was successful and archiving has begun.

Is this possible somehow?

**Comment by pirate** [comment]
You can ctrl+C or send a SIGTERM to the process and it will stop gracefully at any point after parsing.
